lib/msal-common/src/auth/authority/AuthorityFactory.ts

```diff
@@ -20,14 +31,14 @@ export class AuthorityFactory {
         const authorityUrl = new UrlString(authorityString);
         const components = authorityUrl.getUrlComponents();
         const pathSegments = components.PathSegments;
-        switch (pathSegments[0]) {
-            case "tfp":
-                // tfp denotes a b2c url
-                return AuthorityType.B2C;
-            default:
-                // default authority is always AAD
-                return AuthorityType.Aad;
-        }
+
+        if (pathSegments[0].toLowerCase() === "adfs")
```

Comment:
is this a constant for adfs?

Reply:
Yes, confirmed this. But this is also mostly a placeholder until we support adfs

Reply:
Can this string be added to a constants file?

Reply:
Done